### PR TITLE
Bumping the log severity from debug in case of an optimized count() run

### PR DESCRIPTION
**Pull request by davidrabinowitz**
Based on conversation with @emkornfield

### CODE REVIEW

File: connector/src/main/java/com/google/cloud/spark/bigquery/v2/BigQueryDataSourceReader.java [hunk 1]
```diff
@@ -228,6 +232,7 @@ private ReadRowsResponseToInternalRowIteratorConverter createConverter(
 
   List<InputPartition<InternalRow>> createEmptyProjectionPartitions() {
     long rowCount = bigQueryClient.calculateTableSize(tableId, globalFilter);
+    logger.info("Creating a DataFrame of empty rows with size " + rowCount);
```
Comment (emkornfield) [inline connector/src/main/java/com/google/cloud/spark/bigquery/v2/BigQueryDataSourceReader.java:235]: Could we phrase this as something like "Used optimized BQ count(*) path. Count: "

**Review by bsidhom — APPROVED** [review]
LGTM

**Review by emkornfield — APPROVED** [review]
Thanks!

**Pull request merged**